harness/determined/common/experimental/errors.py

```diff
@@ -0,0 +1,4 @@
+class EnterpriseOnlyError(Exception):
```

Comment:
Why not put this in `det.errors`?

Reply:
we have it in the CLI errors: https://github.com/determined-ai/determined/blob/master/harness/determined/cli/errors.py#L1
we should promote that class to `det.errors` and reuse it